DuckDuckGo/UserBehaviorMonitor.swift

```diff
@@ -109,7 +120,11 @@ final class AppUserBehaviorMonitor {
         let domainEvent: Pixel.Event
         switch event {
         case .reloadTwiceWithin12Seconds: domainEvent = .userBehaviorReloadTwiceWithin12Seconds
+        case .reloadTwiceWithin24Seconds: domainEvent = .userBehaviorReloadTwiceWithin24Seconds
+        case .reloadAndRestartWithin30Seconds: domainEvent = .userBehaviorReloadAndRestartWithin30Seconds
+        case .reloadAndRestartWithin50Seconds: domainEvent = .userBehaviorReloadAndRestartWithin50Seconds
         case .reloadThreeTimesWithin20Seconds: domainEvent = .userBehaviorReloadThreeTimesWithin20Seconds
+        case .reloadThreeTimesWithin40Seconds: domainEvent = .userBehaviorReloadThreeTimesWithin40Seconds
```

Comment:
can be reset to what it was, again userBehaviorReloadTwiceWithin12Seconds and userBehaviorReloadThreeTimesWithin20Seconds